<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VEDAMainWindow</class>
 <widget class="QMainWindow" name="VEDAMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VEDAMainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>添加</string>
     </property>
     <addaction name="InitProcessBtn"/>
     <addaction name="separator"/>
     <addaction name="ImportProcessBtn"/>
    </widget>
    <addaction name="InitProjectBtn"/>
    <addaction name="OpenProjectBtn"/>
    <addaction name="separator"/>
    <addaction name="menu_3"/>
    <addaction name="separator"/>
    <addaction name="CloseBtn"/>
    <addaction name="CloseProjectBtn"/>
    <addaction name="separator"/>
    <addaction name="SaveBtn"/>
    <addaction name="SaveAllBtn"/>
    <addaction name="separator"/>
    <addaction name="ExitBtn"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="AboutBtn"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>生成</string>
    </property>
    <addaction name="ExportReportBtn"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_4"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="InitProjectBtn">
   <property name="text">
    <string>新建实验</string>
   </property>
  </action>
  <action name="OpenProjectBtn">
   <property name="text">
    <string>打开实验</string>
   </property>
  </action>
  <action name="CloseBtn">
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
  <action name="CloseProjectBtn">
   <property name="text">
    <string>关闭实验</string>
   </property>
  </action>
  <action name="SaveBtn">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="SaveAllBtn">
   <property name="text">
    <string>保存全部</string>
   </property>
  </action>
  <action name="ExitBtn">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="AboutBtn">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="InitProcessBtn">
   <property name="text">
    <string>新建实验批次</string>
   </property>
   <property name="toolTip">
    <string>新建实验批次</string>
   </property>
  </action>
  <action name="ImportProcessBtn">
   <property name="text">
    <string>现有实验批次</string>
   </property>
   <property name="toolTip">
    <string>现有实验批次</string>
   </property>
  </action>
  <action name="ExportReportBtn">
   <property name="text">
    <string>生成实验报告</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
